### ISSUE TITLE
KUBECONFIG ENV NOT SET 

### ISSUE THREAD

**Issue by zhengbingxian** [chaos-mesh/chaos-mesh#3555]
## Bug Report

**What version of Kubernetes are you using?**

**What version of Chaos Mesh are you using?**

2.3.0

**What did you do?**

run command as follows
```shell
make e2e
```

**What did you expect to see?**
run with no error

**What did you see instead?**
KUBECONFIG ENV NOT SET 。 

Is it a bug?  or it  need to add value  "kubeconfig=/<root>/.kube/config"   as default  value  instead of  to 
run command in most of the scenes and  clusters 。
```shell
export KUBECONFIG=/root/.kube/config
```
If this's not a bug or a optimization of item，i'll close it.

**Comment by Andrewmatilde** [comment]
Hi @zhengbingxian ! It seems run-e2e.sh will set `KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}`. It is strange that KUBECONFIG not set here.

**Comment by zhengbingxian** [comment]
> Hi @zhengbingxian ! It seems run-e2e.sh will set `KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}`. It is strange that KUBECONFIG not set here.

I'cant find the code to call "run-e2e.sh" file in Makefile with the command "make e2e"。 But see some code in  hack/e2e.sh。

- If this is my e2e test usage wrong  by "make e2e"。  
- If the  way to run e2e test  is   access the command "./hack/e2e.sh" only?   if this is right，the provider cluster is only kind, gke, eks， without selection to run local cluster。

**Comment by Andrewmatilde** [comment]
Yes, I agree with you. It seems Makefile cannot access to `run-e2e.sh`. The common method of running e2e test is just `make e2e` which is strange that `make e2e` works well in my env & I have not set the KUBECONFIG too.

**Comment by Andrewmatilde** [comment]
This logic here is confusing now and we are planning to make it clear.

**Comment by zhengbingxian** [comment]
@Andrewmatilde 

maybe kubeconfig is set in your cluster somewhere? so that you can get result by echo :
```shell
$ echo $KUBECONFIG
```

**Comment by Andrewmatilde** [comment]
![image](https://user-images.githubusercontent.com/32479514/185356942-621182ef-85aa-4aa3-b76a-423b3bdd2076.png)
